### ISSUE TITLE
Error after fresh install

### ISSUE THREAD

**Issue by renatodeleao** [whitesmith/qnorr#26]
Using `node 6.10.0`.

Log:
```zsh
➜  cenas git:(master) ✗ npm start

> cenas@0.0.1 start /Users/Renato/github/qnorr_testing/cenas
> node ./node_modules/gulp/bin/gulp.js

[14:32:40] Requiring external module babel-register
[14:32:43] Using gulpfile ~/github/qnorr_testing/cenas/gulpfile.babel.js
[14:32:43] Starting 'default'...
[14:32:43] Starting 'clean'...
[14:32:43] Finished 'clean' after 8.64 ms
[14:32:43] Starting 'vectors'...
[14:32:43] Starting 'images'...
[14:32:43] Starting 'fonts'...
[14:32:43] Starting 'styles'...
[14:32:43] Starting 'scripts'...
[14:32:43] Starting 'views'...
[14:32:44] 'views' errored after 712 ms
[14:32:44] ReferenceError: document is not defined
    at Object.<anonymous> (/Users/Renato/github/qnorr_testing/cenas/src/scripts/helpers/helloWorld.js:1:1)
    at Module._compile (module.js:570:32)
    at loader (/Users/Renato/github/qnorr_testing/cenas/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/Renato/github/qnorr_testing/cenas/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Panini.module.exports [as loadData] (/Users/Renato/github/qnorr_testing/cenas/node_modules/panini/lib/loadData.js:21:14)
    at Panini.module.exports [as refresh] (/Users/Renato/github/qnorr_testing/cenas/node_modules/panini/lib/refresh.js:10:8)
    at module.exports (/Users/Renato/github/qnorr_testing/cenas/node_modules/panini/index.js:41:12)
    at views (/Users/Renato/github/qnorr_testing/cenas/gulpfile.babel.js:190:11)
    at bound (domain.js:280:14)
    at runBound (domain.js:293:12)
    at asyncRunner (/Users/Renato/github/qnorr_testing/cenas/node_modules/async-done/index.js:36:18)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickDomainCallback (internal/process/next_tick.js:122:9)
[14:32:44] 'default' errored after 738 ms
[14:32:44] The following tasks did not complete: vectors, images, fonts, styles, scripts
[14:32:44] Did you forget to signal async completion?

npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/Renato/.nvm/versions/node/v6.10.0/bin/node" "/Users/Renato/.nvm/versions/node/v6.10.0/bin/npm" "start"
npm ERR! node v6.10.0
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! cenas@0.0.1 start: `node ./node_modules/gulp/bin/gulp.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cenas@0.0.1 start script 'node ./node_modules/gulp/bin/gulp.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the cenas package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./node_modules/gulp/bin/gulp.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs cenas
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls cenas
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/Renato/github/qnorr_testing/cenas/npm-debug.log
```

**Comment by Budgerous** [comment]
Just to add some info, this is happening because (somehow) the `views` task is accessing the `helpers` from `scripts/helpers`, even though the path looks well defined

**Comment by mgontav** [comment]
Here is your offending line - https://github.com/whitesmith/qnorr/blob/master/blueprints/default/files/gulpfile.babel.js#L195 . Removing the path to the app root works, so it accessing the app.js file which is already created at the time, which lives at the root.

**Comment by renatodeleao** [comment]
@mgontav thanks. I wonder how that line got there in the first place. Well I'll push it to master and tell our summersmiths to install it via github and you republish this when we got back.

Enjoy Uncle Sam's Wonders!

closed by #27
